test/db/formats/dwarf

```diff
@@ -701,23 +701,23 @@ line: 3
 addr: 0x00001149
 EOF
 EXPECT_ERR=<<EOF
-DEBUG: [cbin.c:3284] (section .dynstr) Css 141 @ 0x480
-DEBUG: [cbin.c:3284] (section .rela.dyn) Cd 8[24] @ 0x550
-DEBUG: [cbin.c:3284] (section .rela.plt) Cd 8[3] @ 0x610
-DEBUG: [cbin.c:3284] (section .init_array) Cd 8[1] @ 0x3db8
-DEBUG: [cbin.c:3284] (section .fini_array) Cd 8[1] @ 0x3dc0
-DEBUG: [cbin.c:3284] (section .dynamic) Cd 8[62] @ 0x3dc8
-DEBUG: [cbin.c:3284] (section .dynstr) Css 141 @ 0x480
-DEBUG: [cbin.c:3284] (section .rela.dyn) Cd 8[24] @ 0x550
-DEBUG: [cbin.c:3284] (section .rela.plt) Cd 8[3] @ 0x610
-DEBUG: [cbin.c:3284] (section .init_array) Cd 8[1] @ 0x3db8
-DEBUG: [cbin.c:3284] (section .fini_array) Cd 8[1] @ 0x3dc0
-DEBUG: [cbin.c:3284] (section .dynamic) Cd 8[62] @ 0x3dc8
-DEBUG: [cbin.c:2552] Cannot resolve symbol address __libc_start_main
-DEBUG: [cbin.c:2552] Cannot resolve symbol address _ITM_deregisterTMCloneTable
-DEBUG: [cbin.c:2552] Cannot resolve symbol address __gmon_start__
-DEBUG: [cbin.c:2552] Cannot resolve symbol address _ITM_registerTMCloneTable
-DEBUG: [cbin.c:2552] Cannot resolve symbol address __cxa_finalize
+DEBUG: [cbin.c:3269] (section .dynstr) Css 141 @ 0x480
```

Comment:
i hate this test :D

Reply:
It was suposed to ensure the log apis work as expected but maybe better to do this in a unit test that we dont need to modify that often